### PR TITLE
Updated Register page

### PR DESCRIPTION
**Pull request by localastronaut**
Added a description to help people know they can sign up using oAuth

### CODE REVIEW

File: templates/registration/register.html [hunk 1]
```diff
@@ -7,15 +7,16 @@ <h2 class="extra-padding">Register</h2>
         <hr>
     </div>
     <div class="center">
-        <a href="{% url 'social:begin' backend='facebook-register' %}"><img class="oauth-icon" src="{% static "img/facebook-signin.png" %}" alt="Register with Facebook" /></a><br/>
-        <a href="{% url 'social:begin' backend='google-oauth2-register' %}"><img class="oauth-icon" src="{% static "img/google-signin.png" %}" alt="Register with Google" /></a>
+        <p>You can sign up with one click using facebook or google</p>
```
Comment (RevolutionTech) [inline templates/registration/register.html]: Suggested copy change: "Sign up with one click using Facebook or Google".

File: templates/registration/register.html [hunk 2]
```diff
@@ -7,15 +7,16 @@ <h2 class="extra-padding">Register</h2>
...
+        <a href="{% url 'social:begin' backend='facebook-register' %}"><img class="oauth-icon" src="{% static "img/facebook-signup.png" %}" alt="Register with Facebook" /></a><br/>
+        <a href="{% url 'social:begin' backend='google-oauth2-register' %}"><img class="oauth-icon" src="{% static "img/google-signup.png" %}" alt="Register with Google" /></a>
     </div>
     <div class="form-input">
+        <p>or create an account</p>
```
Comment (RevolutionTech) [inline templates/registration/register.html]: Suggested copy change: "or create an account the old-fashioned way".

File: templates/registration/register.html [hunk 3]
```diff
@@ -7,15 +7,16 @@ <h2 class="extra-padding">Register</h2>
...
         <form action="" method="POST">
             {% csrf_token %}
             {{ form.as_p }}
             <input class="button submit-button" type="submit" value="Register" />
         </form>
-        <p>...or just <a data-open="login-modal">login</a>.</p>
```
Comment (RevolutionTech) [inline templates/registration/register.html]: Agree with removing this link, that just makes things confusing.

**Comment by RevolutionTech** [comment]
The asset you've added for Google Sign-in is their old branding. We should use the new branding with the white background.

File: templates/registration/register.html (hunk 2 above)
Comment (localastronaut) [inline templates/registration/register.html]: Cool. I like that way better

Comment (localastronaut) [inline templates/registration/register.html]: -- not sure if you saw my comment above -- wasn't sure how to respond to your post about new new google logo so I just edited your comment ha

**Comment by RevolutionTech** [comment]
Sorry, I wasn't notified when you edited my comment. Here is what you're looking for: https://developers.google.com/identity/branding-guidelines

**Comment by RevolutionTech** [comment]
Reviewing the PR again, I also just noticed that you added 4 images here, but meant to only add two. You should remove `icons/facebook-signup.png` and `signup-google.png`.

**Comment by localastronaut** [comment]
Woah... just now seeing this comment. Not sure why I didn't see the e-mail. Just fixed

**Comment by RevolutionTech** [comment]
I don't see any new commits. Maybe you didn't push?

**Comment by localastronaut** [comment]
Ahh... just pushed

**Pull request merged**